heron/proto/packing_plan.proto

```diff
@@ -0,0 +1,31 @@
+package heron.proto.system;
+
+option java_package = "com.twitter.heron.proto.system";
+option java_outer_classname = "PackingPlans";
+
+// This file defines the packing plan of a topology.
+// It reflects the the instances and containers of a topology.
+
+message Resource {
+  required double cpu = 1;
+  required int64 ram = 2;
+  required int64 disk = 3;
+}
+
+message InstancePlan {
+  required string id = 1;
+  required string component_name = 2;
+  required Resource resource = 3;
+}
+
+message ContainerPlan {
+  required string id = 1;
+  repeated InstancePlan instance_plans = 2;
+  required Resource resource = 3;
+}
+
+message PackingPlan {
+  required string id = 1;
+  repeated ContainerPlan container_plans = 2;
+  required Resource resource = 3;
```

Comment:
Yes. This class is basically a protobuf version of [c.t.h.spi.packing.PackingPlan](https://github.com/twitter/heron/blob/master/heron/spi/src/java/com/twitter/heron/spi/packing/PackingPlan.java#L20).